istio/oidc-authservice/overlays/ibm-storage-config/deployment.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: authservice
+spec:
+  template:
+    spec:
+      initContainers:
+      - name: fix-permission
+        image: busybox
```

Comment:
Is this image referenced in kustomization.yaml for the overlay? We do this to ensure that a different container repository can be used.

Reply:
I committed the below section in the kustomization.yaml. We should able to change it to other container repository now.
```
images:
  - name: busybox
    newTag: "latest"
    newName: busybox
```